test/integration/fetch-url-test.js

```diff
@@ -40,4 +40,4 @@ test('account.fetch() and account.profil.fetch()', function (t) {
 
   .catch(t.fail)
 })
-*/
\ No newline at end of file
+*/
```

Comment:
I think this PR fixes https://github.com/hoodiehq/hoodie-client-account/issues/64, right? So we can uncomment this while at it?